http.go

```diff
@@ -1,14 +1,23 @@
 package main
 
 import (
+	"fmt"
 	"log"
 	"net/http"
+	"os"
 	"sync"
 	"time"
 
 	auth "github.com/heroku/lumbermill/Godeps/_workspace/src/github.com/heroku/authenticater"
+	influx "github.com/heroku/lumbermill/Godeps/_workspace/src/github.com/influxdb/influxdb-go"
 )
 
+var influxDbStaleTimeout = 24 * time.Minute // Would be nice to make this smaller, but it lags due to continuous queries.
+var influxDbSeriesCheckPrefixes = []string{
+	"MaxMean1mLoad.10m.dyno.dyno.load.",
```

Comment:
So, this ties this to our implementation of the down stream continuous queries that we've currently defined. I guess this is the right thing to do (i.e. we want to make sure the continuous queries are functioning properly).

Maybe make these definable in a config var so any other users could also use this with their own continuous queries?

Maybe that's all just a premature optimization of a use case that doesn't exist though?